core/domain/opportunity_services.py

```diff
@@ -181,12 +165,34 @@ def _create_exploration_opportunity_summary(topic, story, exploration):
         opportunity_domain.ExplorationOpportunitySummary(
             exploration.id, topic.id, topic.name, story.id, story.title,
             story_node.title, content_count,
-            list(incomplete_translation_language_codes), translation_counts,
-            list(language_codes_needing_voice_artists), []))
+            _compute_exploration_incomplete_translation_languages(exploration),
+            translation_counts, list(language_codes_needing_voice_artists), []))
 
     return exploration_opportunity_summary
 
 
+def _compute_exploration_incomplete_translation_languages(exploration):
+    """Computes all languages that are not 100% translated in an exploration.
+
+    Args:
+        exploration: Exploration. The exploration for which to compute
+            incomplete translation languages.
+
+    Returns:
+        list(str). List of incomplete translation language codes.
+    """
+    audio_language_codes = set(
+        language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES)
+    complete_translation_languages = set(
+        exploration.get_languages_with_complete_translation())
+    incomplete_translation_language_codes = (
+        audio_language_codes - complete_translation_languages)
+    # Remove exploration language from incomplete translation languages list
+    # as an exploration does not need a translation in its own language.
+    incomplete_translation_language_codes.discard(exploration.language_code)
+    return list(incomplete_translation_language_codes)
```

Comment:
Should we sort this list (just for consistency)?

Reply:
To be consistent with constants.SUPPORTED_AUDIO_LANGUAGES?

Reply:
No, more so that the list is generated deterministically each time.

Reply:
Ah, gotcha. Updated.